lib/screens/home/home_bloc.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:async';
+
+import 'package:bloc/bloc.dart';
+import 'package:flutter/material.dart';
+import 'package:pub_client/pub_client.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:tavern/screens/bloc.dart';
+
+class HomeBloc extends Bloc<HomeEvent, HomeState> {
+  @override
+  HomeState get initialState => InitialHomeState();
+  final _htmlParsingClient = PubHtmlParsingClient();
```

Comment:
might want to inject this instead of instantiating it directly in the bloc for testability.

Reply:
What's the benefit for tests?

Reply:
Done